docs/skopeo.1.md

```diff
@@ -114,6 +114,15 @@ Print the version number
 | [skopeo-standalone-verify(1)](skopeo-standalone-verify.1.md)| Verify an image signature.                                   |
 | [skopeo-sync(1)](skopeo-sync.1.md)| Synchronize images between registry repositories and local directories.                |
 
+## EXIT STATUS
+`skopeo` exists with status 0 on success, non-zero on error.
+
+Details about the exist statuses:
```

Comment:
Thanks! LGTM, one last nit:

`s/exist/exit/g`, in both places please

Reply:
Thank you, nice catch! Fixed now too :)

Reply:
I also tweaked the commit message a little bit, hope that was okay too.